frontend/server/src/DAO/Problems.php

```diff
@@ -1438,6 +1438,8 @@ final public static function byIdentityTypeForTypeahead(
                         p.{$searchType} = ?";
         } else {
             $args = array_fill(0, 5, $query);
+            $args[] = preg_replace('/[\W]/', ' ', $query);
+            $args[] = preg_replace('/[\W]/', ' ', $query);
```

Comment:
```suggestion
            $args[] = preg_replace('/\W+/', ' ', $query);
```

no hay necesidad de correr esto dos veces

Reply:
No había necesidad de evaluarlo, pero si de agregarlos. Ya lo arreglé, a ver si así está bien.